docs/_data/table/master-configuration.csv

```diff
@@ -15,6 +15,7 @@ alluxio.master.lineage.recompute.interval.ms,600000
 alluxio.master.lineage.recompute.log.path,${alluxio.logs.dir}/recompute.log
 alluxio.master.port,19998
 alluxio.master.retry,29
+alluxio.master.startup.consistencycheck.enabled,true
```

Comment:
Is there an established convention we are using? We have things like `tieredstore` and `writetype`.